src/main/java/attendance/model/AttendanceWarningLevel.java

```diff
@@ -0,0 +1,38 @@
+package attendance.model;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+public enum AttendanceWarningLevel {
+
+    CLEAN((totalAbsentCount) -> totalAbsentCount < 2, 0),
+    WARNING((totalAbsentCount) -> totalAbsentCount == 2, 1),
+    MEETING((totalAbsentCount) -> totalAbsentCount >= 3 && totalAbsentCount <= 5, 2),
```

Comment:
이런 케이스는 사소할수 있지만 개인적으로 아래와 같이 표현하는게 조건식이 조금 더 잘 읽히더라구요.

```suggestion
    MEETING((totalAbsentCount) -> 3 <= totalAbsentCount && totalAbsentCount <= 5, 2),
```

Reply:
확실히 위 코드가 더 잘 읽히네요:) 해당 부분도 리팩터링 하겠습니다!